solutions/remotemonitoring/armtemplates/basic-parameters.json

```diff
@@ -32,7 +32,7 @@
     "pcsReleaseVersion": {
       "value": "{Added through command line interface}"
     },
-    "dockerTag": {
+    "pcsDockerTag": {
```

Comment:
I'm assuming this string is used in the deployment specifically for RM somewhere?

Reply:
Yes this is specific for RM.